src/main.js

```diff
@@ -140,37 +140,51 @@ var manifestor = function(options) {
           renderLayout(layout.intermediate(), false);
       };
       renderLayout(layout.intermediate(), true, endCallback);
-    } else if (userState.perspective === 'overview' && userState.previousPerspective === 'detail'){
+    } else if (userState.perspective === 'overview' && userState.previousPerspective === 'detail') {
         endCallback = function() {
-        renderLayout(layout.overview(), false);
+        renderLayout(layout.overview(), true);
```

Comment:
The "trues" and "falses" here are the "animate" flag for the layout rendering. If "true" it will animate, if false, it will make the change instantaneously. This is done differently, and in a different order, for the different transitions.